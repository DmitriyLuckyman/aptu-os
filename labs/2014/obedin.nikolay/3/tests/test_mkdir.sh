#!/usr/bin/env bash

source $(dirname $0)/common.sh

test() {
    run ./mkdir     $FS_DIR /test
    run ./ls        $FS_DIR /
    run ./ls        $FS_DIR /test
    run ./mkdir     $FS_DIR /test/test
    run ./mkdir     $FS_DIR /test

    run ./import    $FS_DIR $FILES_DIR/file1 /file1
    run ./ls        $FS_DIR /
    run ./mkdir     $FS_DIR /file1
    run ./ls        $FS_DIR /test

    run ./mkdir     $FS_DIR /
    run ./mkdir     $FS_DIR /very_long_name_here
    run ./mkdir     $FS_DIR /test/test/test/very_long_name_here
    run ./ls        $FS_DIR /test
}
