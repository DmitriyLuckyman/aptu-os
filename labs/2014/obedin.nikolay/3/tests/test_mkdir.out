D test
D test
F file1
Error: file with the same name already exists
Error: empty destination filename
Error: destination filename is too long
Error: destination filename is too long
